src/Models/ProjectLabels.coffee

```diff
@@ -0,0 +1,31 @@
+BaseModel = require '../BaseModel'
+Utils = require '../Utils'
+
+class ProjectLabels extends BaseModel
+  list: (projectId, params = {}, fn = null) =>
+    @debug "ProjectLabels::labels()"
+
+    if 'function' is typeof params
+      fn = params
+      params = {}
+    params.page ?= 1
+    params.per_page ?= 100
+
+    do (->
```

Comment:
Cool. The change is easy, here it is:

https://github.com/node-gitlab/node-gitlab/commit/698d5d7970fae9f9b1e4522acd029f3662b965aa